### PR TITLE
[stable/gocd] Enable use of host aliases

### PR DESCRIPTION
**Pull request by rcjames**
**What this PR does / why we need it**:
Enables use of [HostAliases](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#hostalias-v1-core)

### CODE REVIEW

**Comment by rcjames** [comment]
I believe that I have signed the CLA and appropriately linked it to my account now. Can this be re-checked?

**Comment by rcjames** [comment]
Hello @mattfarina 

I'm not quite sure why @varshavaradarajan was unassigned from this PR, as I was just following the instructions left for me by the bot. Are you able to provide additional information on how I can get this PR moving?  As far as I can tell I am following all the instructions and guidelines which have been provided.

Thanks

**Comment by GaneshSPatil** [comment]
@rcjames -- Thanks for the PR.
either I or @varshavaradarajan will take a look at this PR in sometime.

File: stable/gocd/values.yaml
Comment (varshavaradarajan) [inline stable/gocd/values.yaml]: The convention is to document the usage of this property.  See other properties.

**Review by varshavaradarajan — CHANGES_REQUESTED** [review]
Please update the readme and the changelog as well. One of us will test this out and leave our review. Thanks. :)

**Comment by rcjames** [comment]
Thank you both for your responses.

@varshavaradarajan - I have added documentation to values.yaml and updated the README to list the new property. I have additionally update the version and update the changelog, but I won't know the commit reference until after this PR has been merged. Is that correct?

**Comment by GaneshSPatil** [comment]
@rcjames -- you can make another commit on top of this one for changelog updates. Use the current commit reference in the changelog.

**Comment by mattfarina** [comment]
@arvindsv @ganeshspatil @ganeshspatil @varshavaradarajan Does this look ok?

**Comment by mattfarina** [comment]
This still needs a /lgtm from one of the owners

**Comment by GaneshSPatil** [comment]
@rcjames -- can we resolve the conflicts and make another commit on top of this one for changelog updates. Use the latest commit reference in the changelog.

**Comment by rcjames** [comment]
@GaneshSPatil - I've resolved the merge conflicts and updated the changelog. Let me know if anything else looks amiss.

**Comment by arvindsv** [comment]
Thank you veru much, @rcjames!

**Pull request merged**
